src/builtin.c

```diff
@@ -1778,9 +1763,13 @@ static jv f_strftime(jq_state *jq, jv a, jv b) {
 #endif
   jv_free(b);
   /* POSIX doesn't provide errno values for strftime() failures; weird */
-  if (n == 0 || n > alloced)
+  if ((n == 0 && *fmt) || n > max_size) {
```

Comment:
Ugg `strftime` error handling seems like a mess 😬 so n == 0 might mean error but we only conceder it an error if the format string is not empty? reading the man page it seems %p usage might also return 0?

Reply:
Year, ugly. Hope we can handle error properly but it does not set errno. I think using %p only for the format is very rare use case.